packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -1,6 +1,704 @@
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  replicas: [
+    { region: 'us-east-1' },
+    { region: 'us-east-2' },
+  ],
+});
+```
+
+Alternatively, you can add new replicas to a `GlobalTable` using the `addReplica` method:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  replicas: [{ region: 'us-east-1' }],
+});
+
+globalTable.addReplica({ region: 'us-east-2', deletionProtection: true });
+```
+
+The following properties are configurable on a per-replica basis, but will be inherited from the `GlobalTable` properties if not specified:
+* contributorInsights
+* deletionProtection
+* pointInTimeRecovery
+* tableClass
+* readCapacity (only configurable if the `GlobalTable` billing mode is `PROVISIONED`)
+* globalSecondaryIndexes (only `contributorInsights` and `readCapacity`)
+
+The following example shows how to define properties on a per-replica basis:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  contributorInsights: true,
+  pointInTimeRecovery: true,
+  replicas: [
+    {
+      region: 'us-east-1',
+      tableClass: dynamodb.TableClass.STANDARD_INFREQUENT_ACCESS,
+      pointInTimeRecovery: false,
+    },
+    {
+      region: 'us-east-2',
+      contributorInsights: false,
+    },
+  ],
+});
+```
+
+You can retrieve a single replica from a `GlobalTable` using the `replica` method:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack', { env: { region: 'us-west-2' } });
+
+const globalTable = new dynamodb.GlobalTable(stack, 'GlobalTable', {
+  partitionKey: { name: 'pk', type: dynamodb.AttributeType.STRING },
+  replicas: [
+    { region: 'us-east-1' },
+    { region: 'us-east-2' },
+  ],
+});
+
+const replicaTable = globalTable.replica('us-east-2');
+```
+
+Note: You can create a new `GlobalTable` with as many replicas as needed as long as there is only one replica per region. After table creation you can add or remove replicas, but you can only add or remove a single replica in each update.
+
+## Capacity
```

Comment:
Billing and capacity should be combined into one section, and billing should be introduced first. 

The tree goes:

* OnDemand
* Provisioned
  * Fixed
  * AutoScaled

Reply:
Good call. I've updated the `README` accordingly